schema/site.schema.json

```
@@ -920,6 +920,13 @@
       "examples": [50.0, 0.5],
       "!go": { "pointer": true }
     },
+    "insights.commit.indexer.interval": {
+      "description": "The interval (in minutes) at which the commit indexer will check for new commits.",
```

```suggestion
      "description": "The interval (in minutes) at which the insights commit indexer will check for new commits.",
```

We have another team that is going to build a similar thing (that we may end up using), so let's be specific that this only affects the code insights one.

---

Good call!